python/ray/workflow/common.py

```diff
@@ -103,12 +103,19 @@ class WorkflowStaticRef:
     step_id: StepID
     # The ObjectRef of the output.
     ref: ObjectRef
+    # This tag indicates we should resolve the workflow like an ObjectRef, when
+    # included in the arguments of another workflow.
+    _resolve_like_object_ref_in_args: bool = False
```

Comment:
Can I see that updating the tests is the last thing to enable this by default?

Reply:
Yes.